lblod/app-digitaal-loket
# Feature/add step climate reeks 1 opvolgmoment

Added the form for opvolgmoment & extractor for contact field.

*test*
- checkout this branch in app-digitaal-loket and run app & frontend like usual
- create new subsidy with testMode=true
- create new 'Lokaal Energie- en KlimaatPact' subsidy
- fill in pact & voorstel
- in opvolgmoment make sure contactgegevens are pre-filled with the ones in step 1
- compare form to the one in miro
